packages/dropdown/src/components/Dropdown.test.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Enzyme, { mount } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+import Dropdown from "./Dropdown";
+import DropdownItem from "./DropdownItem";
+
+describe("Dropdown", function() {
+  Enzyme.configure({ adapter: new Adapter() });
+
+  test("default handle selection function returns null", function() {
+    const component = renderer.create(
+      <Dropdown align="right">
+        <DropdownItem value={1}>Option 1</DropdownItem>
+        <DropdownItem value={2}>Option 2</DropdownItem>
+      </Dropdown>
+    );
+    let tree = component.toTree();
+
+    expect(tree.props.handleSelection()).toBeFalsy();
```

Comment:
any strong reason to prefer toBeFalsy instead of toBeNull? (just curious)

Reply:
default handleSelection returns false :P

```
 Dropdown.defaultProps = {
  handleSelection: () => false,
  text: "",
  align: "left"
};
```